packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## 4.1.0 - Oct 17, 2022
+
+New features:
+
+- New scenario-based Teams app template `Workflow bot` that enables you to create workflow style Teams application to facilitate users progress through multiple Adaptive Cards via buttons and inputs. Checkout our [documentation](https://aka.ms/teamsfx-card-action-response) for more details.
```

Comment:
Use the new Workflow bot template to create sequential workflows where Adaptive Cards can be returned in response to user input. Learn more about this new template in the [documentation](https://aka.ms/teamsfx-card-action-response).